### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#132109] [GitHub Education] [answered]
### Hi everyone,

my application for student developer pack is approved on 9th of july but it is not giving me access till now its been 5 days 
![Screenshot 2024-07-14 115802](https://github.com/user-attachments/assets/698a3234-2e58-4127-857c-4851bd9fc534)
KINDLY RESOLVE THIS ISSUE QUICKLY

**Comment by Nahidh96** [comment, marked as answer]
dont worry, bro. same thing happened to me, u would definetely get it before 9 days

**Comment by Nahidh96** [comment]
mine told the same thing and i didnt hv the offeres but like on the 8th day i got all the offers

**Comment by Nahidh96** [comment]
bro, could u mark both my answers as the correct answer pls bro, i want to get the github acheivement badge

**Comment by tigran-sargsyan-w** [comment]
please do not worry you will get all the access and benefits maybe due to the large number of applications it may take a little longer but you will definitely get everything
